magento/magento2
# Do not use store code in URLs when site is in single store mode

### Description (*)
We have a store in single store mode. When we do setup:install, the config option `web/url/use_store` is enabled, which causes the store code to appear in the URL, even though there is only one storeview. This change makes sure the storecode does not appear in the URL when a store is in single store mode.

### Manual testing scenarios (*)

1. Set a store with one store view in Single Store Mode (`bin/magento config:set general/single_store_mode/enabled 1`) and enable store codes (`bin/magento config:set web/url/use_store 1`)
2. Visit the homepage
3. You can see the store code in the URL
4. Apply this patch
5. Visit the homepage
6. Store code is not there

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#36831: Do not use store code in URLs when site is in single store mode
